content/manuals/engine/release-notes/28.md

```diff
@@ -23,6 +23,92 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](/reference/api/engine/version-history.md).
 
+## 28.2.0
+
+{{< release-date date="2025-05-27" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 28.2.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A28.2.0)
+- [moby/moby, 28.2.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A28.2.0)
+
+### New
+
+- Add `{{.Platform}}` as formatting option for `docker ps` to show the platform of the image the container is running. [docker/cli#6042](https://github.com/docker/cli/pull/6042)
+- Add support for relative parent paths (`../`) on bind mount sources when using `docker run/create` with `-v/--volume` or `--mount type=bind` options. [docker/cli#4966](https://github.com/docker/cli/pull/4966)
+- CDI is now enabled by default. [moby/moby#49963](https://github.com/moby/moby/pull/49963)
+- Show discovered CDI devices in `docker info`. [docker/cli#6078](https://github.com/docker/cli/pull/6078)
+- Windows: Initial BuildKit support (opt-in with `DOCKER_BUILDKIT=1`). [moby/moby#49740](https://github.com/moby/moby/pull/49740)
```

Comment:
Wondering if this needs some additional wording around it being

- For _windows images_ (although I guess that confusion would only be when users run Docker Desktop, so maybe redundant).
- "work in progress"
- requires the containerd image store (I ... think?)
- requires a manual installation of containerd as runtime

(Suggestions welcome! Just me thinking out loud)

Reply:
Works for me!

Perhaps we should discuss with @colinhemmings as well to see if these are things we want to amplify (blog-post with more details). Although the experience is still quite manual (having to install containerd yourself etc).